sound/soc/sof/intel/byt.c

```diff
@@ -428,6 +428,34 @@ static void byt_set_mach_params(const struct snd_soc_acpi_mach *mach,
 	mach_params->platform = dev_name(dev);
```

Comment:
@ranj063 @eballetbo this patch does more then adding PM callbacks. Care to add more information in the commit message (or even if possible split this in two separate patches).
I am also now looking into PM for i.MX.